dvc/stage.py

```diff
@@ -502,12 +502,11 @@ def is_cached(self):
         return True
 
     @staticmethod
-    def create(repo, **kwargs):
+    def create(repo, accompany_outs=False, **kwargs):
```

Comment:
I was the one to introduce the `kwargs` instead of list of arguments, but it seems to me that I overdone that, arguments required in this method should be present in arguments list and if necessary, passed down. `kwargs` occludes what is necessary for this method.